src/main/target/STM32_UNIFIED/target.h

```diff
@@ -25,7 +25,9 @@
 
 #define USBD_PRODUCT_STRING     "Betaflight STM32F405"
 
+// Additional drivers included for targets with > 512KB of flash
 #define USE_ACCGYRO_BMI270
```

Comment:
In order to make this easier to maintain consistently, should we reshuffle this so that there is one conditional

```
#if defined(STM32F405) || defined(STM32F745)
// Put everything and the kitchen sink here
[...]
#endif 
```

Reply:
Well the entire file is organized with specific sections for each MCU and then a generic section for all MCUs. So the PR is consistent with that. I'm not sure adding a separate section that again defines a list of supported MCUs for the extra drivers makes it any easier to understand. I guess it saves a minimal amount of typing if additional drivers are added. But at the cost of forcing them to be added for all the listed MCUs rather than selectively in each section. For example the proposed alternative wouldn't allow adding a driver only to F405 and not F745 without reverting back to the current approach.